build/arfset.debug.js

```diff
@@ -48,6 +48,7 @@ ARfset.prototype.display = function () {
     document.body.appendChild(this.canvas);
     //var size = this.canvas.width * this.canvas.height * BYTES_PER_ELEMENT;
     var debugBuffer = new Uint8ClampedArray(Module.HEAPU8.buffer, this.frameIbwpointer, this.frameimgBWsize);
+    console.log(debugBuffer);
```

Comment:
this only print a long list of zeros, like the pointer is initialized but not data was passed...

Reply:
this condition is never true?